### PR TITLE
Head buttons

### PR DESCRIPTION
**Pull request by rokoc003**
image
I add and style 4 select buttons between the header and card component

### CODE REVIEW

File: src/components/_selectbuttons.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+.btn-container {
+  margin: 10px 0px 0px 30px;
+  padding: 5px;
+  flex-direction: row;
+}
+
+.btn-container__button-pizza,
+.btn-container__button-salat,
+.btn-container__button-pasta,
+.btn-container__button-vegan {
```
Comment (phlgr) [inline src/components/_selectbuttons.scss]: Why is there a class for every button?

File: src/components/_selectbuttons.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+@use "../variables";
+
+.btn-container {
+  margin: 10px 0px 0px 30px;
+  padding: 5px;
+  flex-direction: row;
```
Comment (phlgr) [inline src/components/_selectbuttons.scss]: Try adding `width: 100%` to `.btn-container`

File: src/components/_selectbuttons.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+.btn-container__button-pizza,
+.btn-container__button-salat,
+.btn-container__button-pasta,
+.btn-container__button-vegan {
+  background-color: variables.$text-accent;
+  border-radius: 7px;
```
Comment (phlgr) [inline src/components/_selectbuttons.scss]: Increase border-radius for a more round appearance

**Comment by rokoc003** [comment]
image
Buttons are more rounded and a width of 100%

image
I also changed the class of the buttons to a common one

**Pull request merged**